pkg/renovate/bump/bump_test.go

```diff
@@ -61,6 +62,47 @@ func TestBump_versions(t *testing.T) {
 
 }
 
+func TestBump_withExpectedCommit(t *testing.T) {
+
+	dir := t.TempDir()
+
+	tests := []struct {
+		name           string
+		newVersion     string
+		expectedCommit string
+	}{
+		{name: "expected_commit.yaml", newVersion: "7.0.1", expectedCommit: "dbd7bc96fd6cd383b8e895dc4a928d808541bb17"},
```

Comment:
Can this just be a single test that starts with what's currently at like 79? If there's only one case here, it's probably not worth the table-driven scaffolding.

Reply:
the test was generated from the IDE and does make it easier for folks adding new cases but TBH I don't mind either way.  Happy to switch it to a single test if you prefer.  FWIW as this is merged I have opened a second PR to discuss as I did noticed that the test data meant the test didn't actually verify the new sha was correct.  https://github.com/chainguard-dev/melange/pull/368